### PR TITLE
use user desplay name in comments

### PR DESCRIPTION
**Pull request by panayotovyura**

### CODE REVIEW

File: src/AppBundle/Entity/Comment.php [hunk 1]
```diff
@@ -55,6 +55,11 @@ class Comment
 
     /**
      * @ORM\Column(type="string")
+     */
+    private $authorDisplayName;
```
Comment (holyspecter) [inline src/AppBundle/Entity/Comment.php]: i'd propose to link comment directly to user entity so we'll be able to get any information about user that we need.

Comment (panayotovyura) [inline src/AppBundle/Entity/Comment.php]: It is good proposal. But this is just fast fix. If I have time, I would work on it.

Comment (Ishugaev) [inline src/AppBundle/Entity/Comment.php]: I propose using this solution right now. Roman do you agree? Could you please create issue for your proposal - we could implement it bit later

Comment (holyspecter) [inline src/AppBundle/Entity/Comment.php]: @Ishugaev, done: #16

**Pull request merged**